src/Workspaces/Core/Portable/Editing/DeclarationModifiers.cs

```diff
@@ -121,6 +124,8 @@ IMethodSymbol or
 
         public bool IsExtern => (_modifiers & Modifiers.Extern) != 0;
 
+        public bool IsRequired => (_modifiers & Modifiers.Required) != 0;
```

Comment:
Is https://github.com/dotnet/roslyn/blob/9d669eff91debe512e5ccb760c4efa73394d8bb9/src/Workspaces/Core/Portable/Shared/Extensions/ISymbolExtensions.cs#L28-L37 outdated?

(Not only for `IsRequired`)

Can we figure a case where it's problematic?

Reply:
While I can certainly update this codepath, I'm not immediately seeing a way to observe the update.

Reply:
@333fred Probably generate overrides can observe it? https://github.com/dotnet/roslyn/blob/00405d11948c7ca2c44b1ab276c4cf977b30a06d/src/Workspaces/Core/Portable/Shared/Extensions/SyntaxGeneratorExtensions.cs#L496-L500

Reply:
AFAICT, it did not, as I didn't update that codepath before writing that test.